solr/solr-ref-guide/modules/query-guide/pages/highlighting.adoc

```diff
@@ -509,6 +509,20 @@ Furthermore, phrases will be highlighted as a whole instead of word by word.  Cu
 +
 If either `hl.usePhraseHighlighter` or `hl.multiTermQuery` are set to `false`, then this setting is effectively `false` no matter what you set it to.
 
+`hl.passageSort`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: `startOffset`
+|===
++
+The UH uses the score to select the candidates for the number of snippets defined by `hl.snippets`,
```

Comment:
```suggestion
The UH calculates a score for each passage, and it uses it to pick the best snippets defined by `hl.snippets`,
```
(Users probably don't know passages have a score; they can't even see it after all)